packages/perseus/src/components/button-group.tsx

```diff
@@ -66,12 +73,13 @@ class ButtonGroup extends React.Component<Props> {
                 <button
                     title={button.title}
                     type="button"
-                    id={"" + i}
```

Comment:
I can't imagine a world where having HTML elements id'd by a single number being useful or correct.  These buttons are being ID'd as `<button id="0" .../>` which is just ripe for ID collisions so I'm 99.99999% that this is never used and legacy!  As soon as we have more than one `<ButtonGroup>` on a page we'd have id collisions. 

Per [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/id):
> The id [global attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes) defines an identifier (ID) which must be unique in the whole document.